### ISSUE TITLE
Add more Chrome URL's to access more functionality.

### ISSUE THREAD

**Issue by PeapGit** [bypassiwastaken/skiovox-helper#26]
There are a lot of chrome OS url's such as [calculator.apps.chrome](url) and [chrome://camera-app/views/main.html](url) to access chrome apps not normally accessible via this exploit. You should also a link to an HTML to go to your school Google Window where all your school extensions so you don't have to redo the exploit. Normally I would have to go to file manager and go to the HTML in the Extensions New-tab folder in main.html.

**Comment by bypassiwastaken** [comment]
camera app is a good idea. dont think i will do the rest of that though
also last time i checked opening camera app crashes the page

**Comment by PeapGit** [comment]
Would it be possible for the extension to add (from apk) a chrome app and launch it?

**Comment by bypassiwastaken** [comment]
no lol, not even close

**Comment by tuta-amb** [comment]
There's also canvas.apps.chrome, calculators.apps.chrome, and cursive.apps.chrome. Also, the camera app doesn't work for me. I think it would be best to add these builtin apps in a sort of dropdown menu maybe?

**Comment by Epicminer256** [comment]
> we should probably organize all of the chrome apps into one separate apps page just so things don't get cluttered if we do end up doing this

Agreed. I would do it like Chrome's default new-tab where all the apps are in a separate menu. Makes the UI more clean and leaves space for core settings.
